### DISCUSSION TITLE
PyPI links for jupyter-environments are dead

### DISCUSSION THREAD

**Discussion by itcarroll** [datalayer/support#6] [General]
Could you update your project on PyPI at https://pypi.org/project/jupyter-environments/ to fix the links to the homepage? Although ... I don't see a similarly named repo here. Is the project archived? Based on the title and tagline alone, I'd guess this project might be filling an important gap in Jupyter land!

**Comment by echarles** [comment]
Hi @itcarroll Thx for shimming here.

Some packages published on pypi have a github private repo. We will update the publication to point to the main Datalayer github organisation, or maybe to a documentation website. For now we have https://jupyter-kernels.datalayer.tech and https://datalayer.tech which are populated, and plan to populate https://jupyter-environments.datalayer.tech ASAP.

> Based on the title and tagline alone, I'd guess this project might be filling an important gap in Jupyter land!

Yes, absolutely, I will post here examples we are cooking.

**Comment by echarles** [comment]
> I'd guess this project might be filling an important gap in Jupyter land!

While working on this, I was curious to learn more from you around the gaps you were mentioning. Could you share a bit more to what you were thinking, that would be super helpful 🙏.

**Reply by itcarroll** [reply]
Oh, just that there is no GUI to assist with managing virtual environment availability as JupyterLab kernels. My current pattern is to create a project environment (e.g. with poetry, hatch, or conda), add ipykernel ([unfortunately](https://discourse.jupyter.org/t/creating-a-new-kernel-without-installing-ipykernel-in-venv/18541)), then make it known to JupyterLab with `(env) $ python -m ipykernel install --user ...`. Seems like you could have one button do all that (maybe except for the first step b/c of all the pitfalls around environment creation).

**Reply by echarles** [reply]
Thx. Have you tried https://github.com/anaconda/nb_conda_kernels? You will see in your JupyterLab launcher the various conda envs which have kernels.

**Reply by itcarroll** [reply]
No, thanks. Although I threw conda in there for generality ... I mostly use virtualenvs. But I'll look at it!
